packages/experimental/Badge/src/PresenceBadge/PresenceBadge.styling.ts

```diff
@@ -10,22 +10,26 @@ export const stylingSettings: UseStylingOptions<PresenceBadgeProps, PresenceBadg
   states: coreBadgeStates,
   slotProps: {
     root: buildProps(
-      (tokens: PresenceBadgeTokens, theme: Theme) => ({
-        style: {
-          ...getBadgePosition(tokens),
-          width: tokens.width,
-          height: tokens.height,
-          display: 'flex',
-          alignItems: 'center',
-          flexDirection: 'row',
-          alignSelf: 'flex-start',
-          justifyContent: 'center',
-          position: 'absolute',
-          backgroundColor: tokens.backgroundColor,
-          ...borderStyles.from(tokens, theme),
-          ...layoutStyles.from(tokens, theme),
-        },
-      }),
+      (tokens: PresenceBadgeTokens, theme: Theme) => {
+        const { width, height, borderWidth } = tokens;
+        const borderGap = borderWidth * 2;
+        return {
+          style: {
+            ...getBadgePosition(tokens),
+            width: (width as number) + borderGap,
```

Comment:
This seems unsafe, is height/width for tokens not the number type?

Reply:
It's ViewStyle['height'] - it's number | string | undefined. I changed it only on number